sdk/log/simple.go

```diff
@@ -5,13 +5,17 @@ package log // import "go.opentelemetry.io/otel/sdk/log"
 
 import (
 	"context"
+	"sync"
 )
 
 // Compile-time check SimpleProcessor implements Processor.
 var _ Processor = (*SimpleProcessor)(nil)
 
 // SimpleProcessor is an processor that synchronously exports log records.
-type SimpleProcessor struct{}
+type SimpleProcessor struct {
+	exporterMu sync.Mutex
```

Comment:
The comment for NewSimpleProcessor is no longer valid.the exporter does not need to be concurrent safe

Reply:
Removed.